api/iteration.go

```diff
@@ -48,15 +96,19 @@ func NewIteration(dir string, filenames []string) (*Iteration, error) {
 	// All the files should be within the exercism path.
 	for _, filename := range filenames {
 		if !iter.isValidFilepath(filename) {
-			return nil, errUnidentifiable
+			// User has run exercism submit in the wrong directory
```

Comment:
This comment should end in a period, per the [CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences) wiki.